django2pydantic/base.py

```
@@ -325,48 +336,31 @@
     *,
-    django_field: Field[Any, Any] | ForeignObjectRel | GenericForeignKey | property,
+    django_field: Field[Any, Any]
+    | ForeignObjectRel
+    | ForeignObject[Model, Model]
+    | property,
     related_django_model_name: str,
     related_schema: type[BaseModel],
 ) -> tuple[Any, FieldInfo]:
-    default = PydanticUndefined
+    default: PydanticUndefined | None = PydanticUndefined  # type: ignore[valid-type]
     # TODO(phuongfi91): Figure out how to annotate this
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)